### PR TITLE
#440 Add a support for the simplest use case of COBOL data format

### PR DESCRIPTION
**Pull request by yruslan**
This implementation is mostly for getting Cobrix dependencies.
COBOL options, like location of a copybook, etc will go to properties of dataset.

### CODE REVIEW

**Comment by yruslan** [comment]
It's only a beginning. It will get much much harder soon :)

File: standardization/src/main/scala/za/co/absa/enceladus/standardization/StandardizationJob.scala [hunk 1]
```diff
@@ -129,7 +129,8 @@ object StandardizationJob {
       val dfReader1 = if (cmd.rowTag.isDefined) dfReader.option("rowTag", cmd.rowTag.get) else dfReader
       val dfReader2 = if (cmd.csvDelimiter.isDefined) dfReader1.option("delimiter", cmd.csvDelimiter.get) else dfReader1
       val dfReader3 = if (cmd.csvHeader.isDefined) dfReader2.option("header", cmd.csvHeader.get) else dfReader2
-      dfReader3
+      val dfReader4 = if (cmd.rawFormat.equalsIgnoreCase("cobol")) applyCobolOptions(dfReader3, cmd) else dfReader3
```
Comment (Zejnilovic) [inline standardization/src/main/scala/za/co/absa/enceladus/standardization/StandardizationJob.scala]: If you look on line 128 you have `val dfReader4` as well as on line 133

File: standardization/src/main/scala/za/co/absa/enceladus/standardization/CmdConfig.scala [hunk 2]
```diff
@@ -158,6 +161,31 @@ object CmdConfig {
...
+        .validate(value =>
+          if (rawFormat.isDefined && rawFormat.get.equalsIgnoreCase("cobol"))
+            success
+          else
+            failure("The --copybook option is supported only for COBOL data format")
+        )
+
+      opt[Boolean]("is-xcom").optional().action((value, config) => {
+        isXcom = value
+        config.copy(cobolOptions = Some(CobolOptions(copybookLocation, isXcom)))
```
Comment (Zejnilovic) [inline standardization/src/main/scala/za/co/absa/enceladus/standardization/CmdConfig.scala]: wouldn't this line as well as the line 170 profit from being at the end of this method and just once?

**Comment by DzMakatun** [comment]
What happens if the copybook does not match menas schema?

**Pull request merged**